dev-python/pytest-describe/pytest-describe-0.11.1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Describe-style plugin for py.test"
+HOMEPAGE="https://github.com/ropez/${PN} https://pypi.org/project/${PN}"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE=""
+
+RDEPEND="
+	>=dev-python/pytest-2.6.0[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}
```

Comment:
Do you really need pytest at build time when not running tests?

Reply:
I'll test and and update accordingly.